src/Test2.java

```diff
@@ -1,5 +1,5 @@
 public class Test2 {
     public static void main(String[] args) {
-        System.out.println("Test 2252");
+        System.out.println("New test case for password functionality");
```

Comment:
need more test cases like this .It is not enough.

Reply:
![Screenshot 2024-03-26 200638](https://github.com/RosanaKmetovikj/LocalGitPractice_B15/assets/164254613/36e6157e-5342-4d91-8085-2280250e0324)